0.10/README.md

```diff
@@ -85,12 +85,23 @@ Repository organization
 Environment variables
 ---------------------
 
-To set environment variables, you can place them as a key value pair into a `.sti/environment`
-file inside your source code repository.
+Application developers can use the following environment variables to configure the runtime behavior of this image:
+
+NAME        | Description
+------------|-------------
+NODE_ENV    | NodeJS runtime mode (default: production)
+DEV_MODE    | When set to "true", use `nodemon` to automatically reload the server while you work (default: false)
```

Comment:
setting this to true also sets NODE_ENV to development if it's not explicitly set.  and leaving it as false sets NODE_ENV to production (again, if it's not explicitly set)